pages/coordinator/userinformation.vue

```diff
@@ -1,12 +1,195 @@
+        <FormField
+          v-model="infoForm.firstName"
+          label="First Name"
+          placeholder="First Name"
+          type="text"
+        />
+        <FormField
+          v-model="infoForm.lastName"
+          label="Last Name"
+          placeholder="Last Name"
+          type="text"
+        />
+        <FormField
+          v-model="infoForm.phoneNumber"
+          label="Phone Number"
+          placeholder="Phone Number"
+          type="number"
+        />
+        <p v-if="userStore.errorCode" class="text-danger-red">
+          {{ userStore.errorCode }}
+        </p>
+        <div class="w-full flex flex-col gap-6 items-center mt-4">
+          <Button
+            button-text="Update"
+            button-color="bg-light-green"
+            text-color="text-white"
+            size="medium"
+          />
+        </div>
+      </form>
+    </div>
+    <div>
+      <form
+        class="p-5 bg-white heading-montserrat border border-light-grey rounded-3xl max-w-lg py-10 mx-4"
+        @submit.prevent="updatePassword"
+      >
+        <h1 class="text-2xl text-center pb-6">Change Password</h1>
+
+        <FormField
+          v-model="passwordForm.currentPassword"
+          label="Current Password"
+          placeholder="Current Password"
+          type="password"
+        />
+
+        <p v-if="userStore.passwordError" class="text-red-500">
+          {{ userStore.passwordError }}
+        </p>
+
+        <FormField
+          v-model="passwordForm.password"
+          label="Password"
+          placeholder="Password"
+          type="password"
+        />
+
+        <p v-if="errorMessage" class="text-red-500">{{ errorMessage }}</p>
+
+        <FormField
+          v-model="passwordForm.confirmPassword"
+          label="Confirm Password"
+          placeholder="Confirm Password"
+          type="password"
+        />
+        <p v-if="userStore.errorCode" class="text-danger-red">
+          {{ userStore.errorCode }}
+        </p>
+        <p v-if="errorMessage2" class="text-red-500">{{ errorMessage2 }}</p>
+
+        <div class="w-full flex flex-col gap-6 items-center mt-4">
+          <Button
+            button-text="Update"
+            button-color="bg-light-green"
+            text-color="text-white"
+            size="medium"
+          />
+        </div>
+      </form>
+    </div>
+    <Notification
+      :modal-visibility="notificationVisibility"
+      :is-success="isSuccess"
+      :body="notificationMessage"
+      @close="
+        () => {
+          notificationVisibility = false;
+          redirect();
+        }
+      "
+    />
   </section>
 </template>
 
 <script setup>
-import { useHead } from "#imports";
+import { ref, onMounted, onUnmounted } from "vue";
+import { useHead, navigateTo } from "#imports";
+import { useUserStore } from "../../stores/user";
 useHead({
   title: "User Information",
 });
+const userStore = useUserStore();
+
+const infoForm = ref({
```

Comment:
```suggestion
const infoForm = reactive({
```
The Vue team recommends using [`reactive()`](https://vuejs.org/api/reactivity-core.html#reactive) for objects instead of `ref()`.

Reply:
If we change ref to reactive here is there a specific reason we aren't doing it on all the others refs in the website?

Reply:
No reason, I thought I would point it out for future PRs.
You don't have to change it if you don't want to.
I've just noticed that much of the website does it despite it not being correct.